integrationtest/engine-performance/src/main/java/org/hibernate/search/engineperformance/Launcher.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.engineperformance;
+
+import org.openjdk.jmh.runner.Runner;
+import org.openjdk.jmh.runner.options.Options;
+import org.openjdk.jmh.runner.options.OptionsBuilder;
+
+/**
+ * Launches all the JMH benchmarks within this project.
+ * <p>
+ * In order to run the benchmarks, do the following:
+ * <ul>
+ * <li>Generate the JMH benchmark classes by running {@code mvn compile -pl performance} from the root dir</li>
```

Comment:
`mvn compile -pl :hibernate-search-engine-performance`; Also the next bullet point is obsolete for HSEARCH,